src/errors/PluginFailedOnStart.js

```diff
@@ -1,8 +1,9 @@
 const WalletLibError = require('./WalletLibError');
 
 class PluginFailedOnStart extends WalletLibError {
-  constructor(pluginType, pluginName, reason = 'unknown') {
+  constructor(pluginType, pluginName, reason = 'unknown', err) {
     super(`Plugin ${pluginName} of type ${pluginType} onStart failed. Reason: ${reason}`);
+    if (err.stack) this.stack = err.stack;
```

Comment:
I think it's a bad idea. What if we just store an error in local property and provide a method `PluginFailedOnStart #getError`